cards.yaml

```diff
@@ -42,6 +42,56 @@ covid19:
     - remembering when "working from home" meant a day off
     - Zoom dropping out every 10 to 15 seconds
 
+popl2021:
```

Comment:
I would add `default: true` here and remove that line from `munihac2020` to avoid accidentally using the wrong cards during the conference.

Reply:
Done thanks! 

Not sure if we are expecting other cards, so you can merge?